util/install.sh

```diff
@@ -180,7 +184,7 @@ function mn_deps {
 
     echo "Installing Mininet core"
     pushd $MININET_DIR/mininet
-    sudo PYTHON=${PYTHON} make install
+    sudo make PYTHON=${PYTHON} install
```

Comment:
Interesting - I should look at this, but I think we might want to set the environment variable and the make variable.

Reply:
Would
`sudo PYTHON=${PYTHON} make PYTHON=${PYTHON} install`
be better ?